amount_generated: 30
feature_extration_method: SIFT
higher_bound: 25
label_matrix:
- [19.79715695692978, 20.055968035722834, 18.146815982101995, 27.58030511600327, 42.25172667874592,
  14.314822869950667, 9.090873334512967, 7.566221587486883, 91.34958044589301, 32.860173626586786,
  13.132861670499377, 15.580415248222726, 22.825041957560646, 11.300677106384022,
  10.370656093774132, 30.790210550854784, 58.538409041415015, 16.62241672440844, 14.139811868657912,
  22.125760262697792, 37.76928159553864, 19.755301889698995, 18.3078492593788, 24.594423894882443,
  18.524489781152948, 13.622388689087463, 14.38465033729697, 25.804013006653616, 32.27281418243601,
  14.111856262772955, 11.132799479801774, 12.034944314265662, 31.370709181109298,
  15.566368557898361, 21.853098126372455, 35.93712231128152, 52.38460982093619, 16.797172548805193,
  7.48945494951617, 8.881210915837583, 128.04901968458302, 39.90206812397563, 12.349600590179577,
  13.636379430958057, 20.957999204583952, 10.657437514191146, 10.916001689174424,
  50.496397697894125, 61.496497082264824, 20.293683740468673, 25.531478705678193,
  51.25178530032116, 69.84613701029605, 32.237803863313246, 13.965116437782118, 20.251789660826734,
  25.286723105867587, 19.769270085029213, 26.167850331024898, 36.15385188786636, 33.53849947225251,
  11.85314807793662, 9.951106356436137, 17.167791842547523, 26.083867441285157, 11.755346082119681,
  10.908979239027573, 26.615241451843275, 47.671438938777825, 26.258776470872952,
  21.587504951993495, 16.328701111193528, 106.4964519791328, 93.48940696453208, 45.78315750877692,
  20.573330038458025, 17.489396399929543, 10.608331152497557, 12.727363472226225,
  23.18879539189785, 27.01384383776306, 48.188727832175076, 72.67825618499774, 76.95095636047688,
  46.181736309323774, 11.559402188358794, 10.45454935259096, 16.531392536676545, 26.083742141826228,
  26.7273287106651, 37.16784093060313, 26.671336987051912, 18.482580711717155, 9.922950224225595,
  15.797134557790436, 28.923029150184256, 15.328671439694014, 9.069806169954065, 9.853090491175136,
  14.769389329686133, 28.643313214489442, 24.9020326900511, 26.36360651035235, 23.88114586177493,
  29.741182830313427, 22.67135790081098, 16.51753244024765, 12.06281333345902, 20.580388205451772,
  23.19585518743308, 38.671243242575095, 40.75531177059642, 26.391606280146615, 22.888109359539097,
  23.86713861527337, 16.9511631569153, 20.972121284785896, 21.818267048643136, 36.86717186796325,
  35.139899962326936, 21.727158686057955, 21.26581974800592, 18.13293988180769, 11.258797482419238,
  13.957947615123384, 14.377639055750386, 26.580463270631736, 28.13287968088819]
- [16.72226939392603, 16.70808829740283, 15.569344645102868, 15.722534763778198, 21.693737136889105,
  21.65064805564932, 19.464152338190743, 17.78933776985492, 38.545427382301284, 23.124391299841417,
  18.32534675567453, 20.445145495856927, 29.44503685080467, 24.68904551236243, 20.751324120190898,
  23.2343921061117, 72.71291199598397, 23.870873888902924, 12.492927387740352, 16.684225982583566,
  17.636339354154686, 14.923407932360037, 19.210528436372865, 35.32527535415226, 21.913960343570707,
  9.086168959603643, 7.779805338578673, 12.933016546842797, 26.9473501527597, 22.67463226770595,
  19.69372712870875, 22.999970017899592, 20.22014062091397, 19.918734996201664, 27.574254942770633,
  27.35882110583027, 23.794335071679516, 31.660266744863215, 24.71294283803786, 18.727276701781538,
  42.52165199933093, 33.31091386150851, 48.18654824723293, 60.04784457001072, 45.69849788110997,
  39.11004059594295, 16.626954037995834, 12.095646758045852, 124.88996679998779, 56.770289747492434,
  24.435437813329568, 17.856294609798763, 14.813367644257399, 10.708063573711938,
  9.459412341103148, 36.00480710010592, 35.296551411599964, 10.320523570833117, 8.779952761638857,
  20.31099957931692, 43.22962595204608, 29.043061660551395, 16.09577984503135, 20.066990876257318,
  17.248949333398652, 20.157797149762832, 27.478459153336303, 35.607718660879776,
  25.36834491250966, 24.16738629609868, 29.626775328717994, 19.770425046018687, 42.73689147007187,
  14.219915869382913, 21.65069796892335, 39.44029184572719, 46.82766592356548, 58.976068594867016,
  46.124318748684935, 32.052768555876, 125.91864089931961, 40.4163463509303, 11.03822804168634,
  10.674636166000246, 17.59330154206465, 16.641156254349724, 19.784686071410032, 54.138698783074396,
  34.68890425614064, 22.49765036291376, 18.60764911905638, 31.04780680080096, 45.88999887246068,
  19.143481907473276, 6.8419682455403175, 9.53114824692524, 16.8804300458223, 15.732111345198303,
  18.06701543414487, 22.2966373389504, 21.172237374267272, 17.631658698246838, 20.64594474095377,
  18.50231713449364, 35.97602081127373, 19.100466606336518, 22.363556313129823, 23.315856185001252,
  33.75113174504474, 25.057380031421218, 19.72726126335572, 24.339655271325366, 72.20105069359066,
  40.73199723282545, 21.909033876956546, 18.598118919597532, 22.29191988047979, 12.51675942128805,
  10.01910101628589, 26.775012669043708, 24.631623328223096, 28.258385874099538, 21.402001107332907,
  21.2440933979419, 29.823076973413063, 9.645912769651401, 6.8995852628522565, 11.291936008152815]
- [11.297363732043138, 21.216217316411193, 28.2501033562408, 25.378459362523007, 26.053972531784222,
  15.540639140303673, 8.020329711489568, 7.770271401181382, 25.966165736221306, 35.810925862670615,
  43.648619996160456, 30.006843652232615, 14.094515370719149, 10.959455782472372,
  15.810841107473548, 20.22304898781475, 30.702644451741865, 35.57422430405605, 36.817668938362964,
  20.803846212555907, 8.371649800854287, 13.303978259788861, 26.54055460020062, 31.27042077013778,
  22.067405289504162, 22.85820973472845, 23.473043264283422, 11.06066831240553, 10.270306238934028,
  12.77025953153281, 24.5066503597515, 30.70267599669736, 19.580880870252575, 11.567569575055932,
  21.03377317581334, 27.29736082039232, 39.8310420412805, 29.628482701447762, 12.871491868647798,
  10.668901444767812, 103.27701839994421, 18.587946388623905, 14.081056234445844,
  12.48639939293326, 14.141870785194026, 22.810857555014042, 47.006590660372396, 92.89166395596489,
  42.58100644849553, 21.608225206330886, 8.918900152244886, 5.641920821432203, 25.30410650428433,
  69.65544983537525, 76.5676158569596, 56.56765917853506, 30.337876585988784, 24.94585109551689,
  14.06768605209686, 8.371683113354749, 17.722878533004902, 29.378425756692028, 39.62832710215292,
  30.162100929971075, 30.43917872883212, 9.04740326630137, 7.885044185067136, 16.500068173662804,
  45.885146770543344, 33.45247854326699, 22.114914246044272, 16.108048295410825, 121.91212746717163,
  40.77023247230982, 3.7162407841506675, 4.972908192786731, 18.810617041694226, 17.750044393510322,
  22.006723865906594, 45.27694041264891, 75.43241149821662, 20.97973882547269, 9.513475263254541,
  14.865045131867701, 50.59455339074998, 50.864894294692704, 33.12157970916468, 31.567564984345346,
  23.44591097547287, 16.34454226234519, 11.432493839451265, 13.972960168580732, 27.594590118695486,
  41.4527568175531, 29.844503623159476, 13.621461782054325, 21.831217604211144, 10.411979473384775,
  8.087907211085994, 14.25659708186285, 35.135078230865034, 21.22975898478948, 19.358077848245532,
  20.601300394702907, 95.84457902600093, 42.682526325327174, 8.783808284473336, 6.364819388632353,
  18.993141587795805, 13.195812935535727, 16.371637259855145, 32.31736639675557, 54.709502431502955,
  36.08090896155119, 17.871485618603817, 14.466251593091473, 35.6147809424753, 17.72975408614333,
  8.689248192304236, 15.175634964186017, 15.513384135047891, 11.70279102643087, 15.135196742121234,
  20.614871552005294, 26.283553736816025, 21.466262059439476, 9.493300775645986, 7.729584329162818]
lower_bound: -5
node_ids: {node0: 0, node1: 1, node2: 2}
num_agents: 3
publish_queue_size: 100
publish_rate: 5
to_ids: {node0: 1, node1: 2, node2: 0}
